jenkins-scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# Jenkins scripts
+
+We use [DSL Jenkins plugin](https://plugins.jenkins.io/job-dsl/) to allow us to manage and define the jobs as human readable code. You can find the different job configs under the [`dsl`](./dsl/) folder. 
```

Comment:
Nitpick: real reason to use the DSL jenkins code is the ability to programmatically generate the job configuration (configuration as code).

Reply:
Nitpick II: I try to avoid the first person when writing docs, something like: "The release-tools repository uses the DSL Jenkins plugin...".